nilearn/plotting/find_cuts.py

```diff
@@ -209,6 +210,13 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
 
     data = _smooth_array(data, affine, fwhm='fast')
 
+    if (isinstance(n_cuts, numbers.Number) and n_cuts > 0):
```

Comment:
Case1:
```python
if isinstance(n_cuts, numbers.Number) and abs(round(n_cuts) - n_cuts) < 1e-10:
    condition
    # Comment: Condition is passing always since I am subtracting n_cuts with n_cuts
```
Narrowing down Case 1
Case2: fix_n_cut = 0
```python
if isinstance(n_cuts, numbers.Number) and abs(round(n_cuts) - fix_n_cut) < 1e-10:
    condition
    # condition passes as expected but fails for cut_coords= -1 
```
Narrowing down Case 2
Case 3: fix_n_cut = 0 and removed ```abs```
```python
if isinstance(n_cuts, numbers.Number) and (round(n_cuts) - fix_n_cut) < 1e-10:
    condition
    # condition always passes as expected for cut_coords which are <=0 
```
@lesteve @GaelVaroquaux Can you comment on this ?